client/src/components/ReviewList.js

```diff
@@ -26,7 +26,7 @@ const ReviewList = ({ reviews, currentUser }) => {
   }
 
   const reviewTiles = reviews.map(review => {
-    return <ReviewTile key={review.id} review={review} currentUser={currentUser} castVote={castVote} />
+    return <ReviewTile key={review.id} review={review} currentUser={currentUser} castVote={castVote} reviews={reviews} restaurant={restaurant} setRestaurant={setRestaurant} />
```

Comment:
components with multiple props should be written on all separate lines for readability
```js
return (
  <ReviewTile
    key={review.id}
    review={review}
    currentUser={currentUser}
    ....
  />
)
```